### PR TITLE
Add dithering effect for players covering biddingplatforms

### PR DESCRIPTION
**Pull request by Fueredoriku**
Now it's easier to read descriptions even when players are standing on top of the biddingplatforms

### CODE REVIEW

File: Assets/Scripts/Auction/BiddingAI.cs [hunk 1]
```diff
@@ -30,6 +30,7 @@ void Start()
         }
         playerManager.onSelectedBiddingPlatformChange += OnBiddingPlatformChange;
         playerManager.onSelectedBiddingPlatformChange += AnimateChipStatus;
+        InstatiateMaterials();
```
Comment (toberge) [inline Assets/Scripts/Auction/BiddingAI.cs]: Insta**n**tiate 😤

File: Assets/Scripts/Auction/BiddingPlayer.cs [hunk 2]
```diff
@@ -20,12 +23,48 @@ public class BiddingPlayer : NetworkBehaviour
...
-    protected BiddingPlatform currentPlatform = null;
+    private BiddingPlatform _currentPlatform = null;
+    protected BiddingPlatform currentPlatform
+    {
+        get => _currentPlatform;
+        set
+        {
+            _currentPlatform = value;
+            instantiatedMaterials
+                .ForEach(material =>
```
Comment (toberge) [inline Assets/Scripts/Auction/BiddingPlayer.cs]: Don't we aleready have an OnBiddingPlatformChange delegate that could be subscribed to for this? Feels a lil arbitrary to leave it in the setter.

File: Assets/Scripts/Auction/BiddingPlayer.cs [hunk 3]
```diff
@@ -20,12 +23,48 @@ public class BiddingPlayer : NetworkBehaviour
...
+            _currentPlatform = value;
+            instantiatedMaterials
+                .ForEach(material => 
+                    material.SetFloat("_DitherDensity", value != null ? 0.9f : 1f));
+        }
+    }
+    [SerializeField]
+    protected SkinnedMeshRenderer[] playerRenderers;
+    [SerializeField]
+    protected MeshRenderer signRenderer;
```
Comment (toberge) [inline Assets/Scripts/Auction/BiddingPlayer.cs]: I think SkinnedMeshRenderer and MeshRenderer both inherit from Renderer if you wanna merge these properties (no biggie though)

**Review by toberge — APPROVED** [review]
Looks and works really good, I just have a few picknits

File: Assets/Scripts/Auction/BiddingAI.cs (hunk 1 above)
Comment (Fueredoriku) [reply Assets/Scripts/Auction/BiddingAI.cs]: Fixed

File: Assets/Scripts/Auction/BiddingPlayer.cs (hunk 2 above)
Comment (Fueredoriku) [reply Assets/Scripts/Auction/BiddingPlayer.cs]: More code, but I did it anyway lol

File: Assets/Scripts/Auction/BiddingPlayer.cs (hunk 3 above)
Comment (Fueredoriku) [reply Assets/Scripts/Auction/BiddingPlayer.cs]: Yeah, too annoying so I fixed

**Pull request merged**